backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -162,50 +142,41 @@ private boolean requireSpecificPatientSearchPermission(
       return List.of(); // empty list
     }
 
-    // first check permissions
-    if (!requireSpecificPatientSearchPermission(facilityId, isArchived, namePrefixMatch)) {
-      throw new AccessDeniedException("Access is denied");
-    }
-
     return _repo.findAll(
         buildPersonSearchFilter(facilityId, isArchived, namePrefixMatch),
         PageRequest.of(pageOffset, pageSize, NAME_SORT));
   }
 
+  @AuthorizationConfiguration.RequireSpecificPatientSearchPermission
   public long getPatientsCount(UUID facilityId, boolean isArchived, String namePrefixMatch) {
-    // first check permissions
-    if (!requireSpecificPatientSearchPermission(facilityId, isArchived, namePrefixMatch)) {
-      throw new AccessDeniedException("Access is denied");
-    }
     if (namePrefixMatch != null && namePrefixMatch.trim().length() < MINIMUM_CHAR_FOR_SEARCH) {
       return 0;
     }
     return _repo.count(buildPersonSearchFilter(facilityId, isArchived, namePrefixMatch));
   }
 
   // NO PERMISSION CHECK (make sure the caller has one!) getPatient()
-  public Person getPatientNoPermissionsCheck(String id) {
+  public Person getPatientNoPermissionsCheck(UUID id) {
     return getPatientNoPermissionsCheck(id, _os.getCurrentOrganization());
   }
 
   // NO PERMISSION CHECK (make sure the caller has one!)
-  public Person getPatientNoPermissionsCheck(String id, Organization org) {
-    UUID actualId = UUID.fromString(id);
```

Comment:
Yes, I'd say it's an improvement when you no longer need the variable _actual_ Id